src/client/components/pages/Courses/NotesModal.tsx

```diff
@@ -1,3 +1,5 @@
+import { HttpStatus } from '@nestjs/common';
```

Comment:
I'd prefer not to use any of the `@nestjs` or server-specific packages in our client-side code, because: 

1. It keeps our frontend app backend-agnostic 
2. It could potentially cause problems down the line if any node-specific code gets added to the package we're importing

There is a standalone [`http-status-codes`][hsc] package we could install, or we can just define our own constant to use in place.


[hsc]: https://www.npmjs.com/package/http-status-codes

Reply:
(I did miss this in the enrollment modal PR. We don't need to update that one now, but we probably should in a future bugfix)

Reply:
Yep. Totally agree with all of this. I added an enum in f6d3fbcbb74040e35a9eb503cc2d583d6b192a01 to take care of this and swapped `@nestjs/common` out for it in 0dd6110a07ffda5668996e8ddbb3cf856464e8aa